.gitignore

```diff
@@ -7,5 +7,3 @@ node_modules/
 *.code-workspace
 .DS_Store
 coverage
-/api-enumerations/
-/dist/
```

Comment:
dist already in here and api-enumerations just gets added as a submodule - git should handle it